Controllers films users

tema3303 opened the pull request.

tema3303 committed:
Сделал 1 вариант 9 ТЗ

tema3303 committed:
Сделал 1 вариант 9 ТЗ

jeggga commented on src/main/java/ru/yandex/practicum/filmorate/validator/ValidationFilms.java:
```diff
@@ -0,0 +1,31 @@
+
+import java.time.LocalDate;
+
+public class ValidationFilms {
```
Отлично, что решил выделить валидацию в отдельный класс

jeggga commented on src/test/java/ru/yandex/practicum/filmorate/validations/FilmTest.java:
```diff
@@ -0,0 +1,71 @@
+
+public class FilmTest {
+    @Test
+    void contextLoads() {
```
Что за пустой тест?

tema3303 committed:
1)Удалил пустой тест
2)Сделал валидацию через аннотации везде, кроме даты релиза. Ее оставил в отдельно классе.
3)Не разобрался, нужны ли тесты на стандартные аннотации, пока удалил. Оставил тест для даты релиза
